### ISSUE TITLE
set_nkpts fails at optimize_calc_options()

### ISSUE THREAD

**Issue by anoopkcn** [JuDFTteam/aiida-fleur#82]
```
02/28/2020 05:15:37 PM <2174> aiida.orm.nodes.process.workflow.workchain.WorkChainNode: [REPORT] [26965|FleurBandDosWorkChain|start]: started bandsdos workflow version 0.3.5
02/28/2020 05:15:37 PM <2174> aiida.orm.nodes.process.workflow.workchain.WorkChainNode: [REPORT] [26965|FleurBandDosWorkChain|run_fleur]: INFO: run FLEUR
02/28/2020 05:15:37 PM <2174> aiida.orm.nodes.process.workflow.workchain.WorkChainNode: [REPORT] [26970|FleurBaseWorkChain|on_except]: Traceback (most recent call last):
  File "/Users/chand/pyenvs/aiida/lib/python3.7/site-packages/plumpy/process_states.py", line 220, in execute
    result = self.run_fn(*self.args, **self.kwargs)
  File "/Users/chand/workbench/devel/aiida-core/aiida/engine/processes/workchains/workchain.py", line 177, in _do_step
    finished, stepper_result = self._stepper.step()
  File "/Users/chand/pyenvs/aiida/lib/python3.7/site-packages/plumpy/workchains.py", line 283, in step
    finished, result = self._child_stepper.step()
  File "/Users/chand/pyenvs/aiida/lib/python3.7/site-packages/plumpy/workchains.py", line 234, in step
    return True, self._fn(self._workchain)
  File "/Users/chand/workbench/devel/aiida-fleur/aiida_fleur/workflows/base_fleur.py", line 139, in validate_inputs
    self.check_kpts()
  File "/Users/chand/workbench/devel/aiida-fleur/aiida_fleur/workflows/base_fleur.py", line 159, in check_kpts
    machines, mpi_proc, omp_per_mpi, self.ctx.use_omp, self.ctx.suggest_mpi_omp_ratio, fleurinp)
  File "/Users/chand/workbench/devel/aiida-fleur/aiida_fleur/tools/common_fleur_wf.py", line 594, in optimize_calc_options
    kpts = int(kpts[0])
IndexError: list index out of range
```

**Comment by anoopkcn** [comment]
PS: There is also a possibility that instead of `/fleurInput/calculationSetup/bzIntegration/kPointList/@count` tag `/fleurInput/calculationSetup/bzIntegration/kPointCount/@count` is given.  Since  I'm  doing a generalised version of setKpointsData #57

**Comment by Tseplyaev** [comment]
Fixed by f9d61bf9556d

**Comment by broeder-j** [comment]
Comment: also there is now a switch in options, `optimize_resources` , which will turn resource optimization off
